src/test/java/uk/gov/pay/ledger/utils/fixtures/EventFixture.java

```diff
@@ -119,4 +121,37 @@ public String getEventType() {
     public String getEventData() {
         return eventData;
     }
+
+    public EventFixture from(Event event) {
+        id = event.getId();
+        sqsMessageId = event.getSqsMessageId();
+        resourceType = event.getResourceType();
+        resourceExternalId = event.getResourceExternalId();
+        eventDate = event.getEventDate();
+        eventType = event.getEventType();
+        eventData = event.getEventData();
+        return this;
+    }
+
+    @Override
+    public EventFixture insert(AmazonSQS sqsClient) {
```

Comment:
Not a big issue but can we separate Queue fixtures to DB fixtures to avoid any confusion